### PR TITLE
Lorem Ipsum generator

### PR DESCRIPTION
**Pull request by jee1mr**
What does your Instant Answer do?
Generated random Lorem Ipsum generator with desired number of paragraphs

What problem does your Instant Answer solve (Why is it better than organic links)?
Open source version of the already existing instant answer

What is the data source for your Instant Answer? (Provide a link if possible)
CPAN module

Why did you choose this data source?
Convenient

Are there any other alternative (better) data sources?
There are many alternative sources: http://loripsum.net/api

What are some example queries that trigger this Instant Answer?
"lorem ipsum", "lipsum", "lorem ipsum 2", "lipsum 10"

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
Web developers, Programmers, Content Writers

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
Solves #958

Which existing Instant Answers will this one supercede/overlap with?
https://duckduckgo.com/?q=lorem+ipsum
https://duckduckgo.com/?q=bacon+ipsum&t=ffab&ia=baconipsum

Are you having any problems? Do you need our help with anything?
Yes. It's not printing in paragraphs but as a single paragraph.

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)

Checklist

Please place an 'X' where appropriate.

- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

### CODE REVIEW

**Comment by jee1mr** [comment]
@mintsoft Since the previous PR had merge conflicts, I made a new one. I will work on the changes you mentioned.

**Comment by mintsoft** [comment]
Follow on from: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1003

**Comment by mintsoft** [comment]
@jee1mr No problem; just ping us when you're ready :+1:

**Comment by mintsoft** [comment]
@jee1mr Are you still interested in continuing on with this?

**Comment by jee1mr** [comment]
@mintsoft Yes. I had some trouble. Can I get some help?

**Comment by jee1mr** [comment]
@mintsoft @moollaza Done with this, I guess. But there's an issue. The output format in the search result is different from what is printed in the console. The paragraphs and extra spaces are getting removed. How to fix that?

**Comment by mintsoft** [comment]
@jee1mr Apologies for the delay; the output format will be different as you need to return html as well as text to get the display like you want. 
```perl
return  answer => $text,
                html => $html;;
```
If you wrap each paragraph a `<p></p>`then you should find the output closer to what you see on the console.

**Comment by jee1mr** [comment]
@mintsoft Should I use a <pre> tag? Or Is it better to wrap them in <p>?

**Comment by jee1mr** [comment]
@mintsoft Thanks :+1: I did that. 
Please review.

**Comment by jee1mr** [comment]
@mintsoft @moollaza ping!

File: lib/DDG/Goodie/LoremIpsum.pm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    $lorem[$i] = "<p>" . $text->paragraphs(1) . "</p>"; 
+}
+
+handle remainder => sub {
+    my $html = "";
+    my $loop;
+    $loop = $_ if $_ && $_ =~ /(^\d{1}$)|10/;
+    $loop = 10 if $_ && $_ =~ /^\d{2,}$/;
+    $loop = 4 if !$_;
+    for my $i (1..$loop){
```
Comment (mintsoft) [inline lib/DDG/Goodie/LoremIpsum.pm]: Can this be simplified a little by doing something like:
```perl
my $loop = 4;
$loop = $_ if $_ && $_ =~ /(^\d+$)/;
$loop = 10 if $loop > 10;
```

**Comment by moollaza** [comment]
@jee1mr sorry about the delay, will get around to this shortly. Looks like there are merge conflicts right now though, please pull in master and resolved the conflicts.

**Comment by jee1mr** [comment]
@moollaza No problem. 
@mintsoft https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1106

**Comment by moollaza** [comment]
Oh -- I didn't mean for you to make a new PR -- but it's no prob, thanks for updating!

**Pull request closed without merging**